joint0_controller:
  type: robot_mechanism_controllers/JointPositionController
  joint: joint0
  pid:
    p: 1.0
    i: 0.4
    d: 0.4

joint1_controller:
  type: robot_mechanism_controllers/JointPositionController
  joint: joint1
  pid:
    p: 1.0
    i: 0.4
    d: 0.4

joint2_controller:
  type: robot_mechanism_controllers/JointPositionController
  joint: joint2
  pid:
    p: 1.0
    i: 0.4
    d: 0.4


joint3_controller:
  type: robot_mechanism_controllers/JointPositionController
  joint: joint3
  pid:
    p: 1.0
    i: 0.4
    d: 0.4

joint4_controller:
  type: robot_mechanism_controllers/JointPositionController
  joint: joint4
  pid:
    p: 1.0
    i: 0.4
    d: 0.4

